### DISCUSSION TITLE
How does this compare to Xournal++?

### DISCUSSION THREAD

**Discussion by kxygk** [LinwoodDev/Butterfly#378] [General]
I don't mean this in any confrontation sense, I just am wondering what differences do the devs see that set this apart from Xournal++? They seem to at first glance be striving to do something similar - but maybe the design goals are quite distinct. Would be nice if this was spelled out somewhere. Thanks! I've only used it a bit, but the app is very slick

**Comment by CodeDoctorDE** [comment]
Hi,
Thanks for your interest.
The biggest difference to xournal++ is the infinite canvas, the customizable painters, and the mobile and web version.
Technically, xournal++ uses gtk while butterfly uses flutter.
Currently, I don't see xournal++ to have active releases

**Comment by kxygk** [comment]
Thanks for the background!

I just noticed it seems to produce pretty clean SVGs on export! There aren't a ton of features (no fills/gradients, line paths, polygons etc) but could be a nice way to start out on a tablet and then do some finishing touches in Inkscape on the computer

Thanks for making this :))

**Reply by CodeDoctorDE** [reply]
Polygons and gradients (#93) are planned, but I'm currently focussing on more important parts of the app.
Line paths are already there, just use the shape painter and select the line shape or (in the 2.0 version) click on the plus icon and on "Shapes" --> "Line"

**Reply by kxygk** [reply]
Right, but you can't have connected line paths - so that line2 starts where line1 ended (into one SVG object). You just draw them one at a time. Maybe you can then somehow glue them together in Inkscape to make a poly - but I'm no expert

**Reply by CodeDoctorDE** [reply]
Hmm, that could be an idea. Feel free to create an issue and I will add it to my to-do list for 2.x releases.

**Reply by kxygk** [reply]
I don't know if in the end you are shooting for a SVG authoring tool! So it's not a criticism - just an observation
